build/nuget-release.yml

```diff
@@ -98,22 +98,22 @@ stages:
             inputs:
               artifact: 'Build'
               path: '$(Build.SourcesDirectory)'
-          - ${{ if not(contains(variables['Build.BuildNumber'], '-')) }}:
-              - template: github/create-release.yml@templates
-                parameters:
-                  repositoryName: 'arcus-azure/arcus.security'
-                  releaseNotes: |
-                    Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Security.All/$(Build.BuildNumber))
-                    ```shell
-                    PM > Install-Package Arcus.Security.All --Version $(Build.BuildNumber)
-                    ```
-          - ${{ if contains(variables['Build.BuildNumber'], '-') }}:
-              - template: create-pre-release.yml@templates
-                parameters:
-                  repositoryName: 'azure-arcus/arcus.security'
-                  releaseNotes: |
-                    Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Security.All/$(Build.BuildNumber))
-                    ```shell
-                    PM > Install-Package Arcus.Security.All --Version $(Build.BuildNumber)
-                    ```
+          - template: github/create-release.yml@templates
+            parameters:
+              repositoryName: 'arcus-azure/arcus.security'
+              releaseNotes: |
+                Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Security.All/$(Build.BuildNumber))
+                ```shell
+                PM > Install-Package Arcus.Security.All --Version $(Build.BuildNumber)
+                ```
+            condition: not(contains(variables['Build.BuildNumber'], '-'))
+          - template: create-pre-release.yml@templates
+            parameters:
+              repositoryName: 'azure-arcus/arcus.security'
+              releaseNotes: |
+                Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Security.All/$(Build.BuildNumber))
+                ```shell
+                PM > Install-Package Arcus.Security.All --Version $(Build.BuildNumber)
+                ```
+            condition: contains(variables['Build.BuildNumber'], '-')
```

Comment:
Does this work, I explicitly removed this because it didn't.

Reply:
I thought I've tried this but can't find it 🤔 

Apparently it doesn't work for templates :/ Will abandon.